creatures/creatures.cpp

```diff
@@ -623,6 +624,56 @@ bool Player::canWear(const std::shared_ptr<Object>&  object, bool all) const {
     return(true);
 }
 
+//********************************************************************
+//                      canUseWands
+//********************************************************************
+bool Player::canUseWands(bool print) {
+    bool useable=false;
+
+    if(isStaff())
+        useable = true;
+
+    switch (getClass()) {
+    case CreatureClass::LICH:
+    case CreatureClass::DRUID:
+    case CreatureClass::CLERIC:        
+    case CreatureClass::MAGE:
+    case CreatureClass::BARD:
+        useable = true;
+        break;
+    case CreatureClass::THIEF:
+        if(getSecondClass() == CreatureClass::MAGE)
+            useable = true;
+        else if(getLevel() >= 10)
+            useable = true;
+        break;
+    case CreatureClass::FIGHTER:
+        if(getSecondClass() == CreatureClass::THIEF && getLevel()>=10)
+            useable = true;
+        if(getSecondClass() == CreatureClass::MAGE)
+            useable = true;
+        break;
+    default:
+        useable = false;
+
+        break;
+    }
+     //Just to remove annoying compiler warning..until we actually use this function
+    if(!useable)
+        useable = false;
```

Comment:
What's this about?

Reply:
It is explained completely by that comment.

Reply:
What is the compiler warning? Unused variable or something? If so could set it to false here and return it to achieve the same w/o check

Reply:
I was half asleep probably when i did that off the cuff. I'll change it.

Reply:
Instead of this weird shit I did, I changed it and put in useable=true after all of the checks, with comments explaining. Function not used yet anyway, so trivial for the moment.